_posts/0000-01-02-first-coding.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
[Markdown](https://guides.github.com/features/mastering-markdown/) is supported.